server/README.md

```diff
@@ -19,7 +19,119 @@ Domain: `covid19app.who.int`
+#### Static Content
+
+Static content is hosted by Firebase Hosting. Everything in the `app/server/public` folder is hosted. Most notable is the `content` folder; the contents of that folder are automatically produced at deploy-time.
+
+Static content will be available via a URL like `https://OUR-PROJECT-ID.web.app` as well as any of our custom domains that we attach to the project (e.g. `example.whocoronavirus.org`).
+
+#### Dynamic content, HTTPS APIs, server-side code
+
+Dynamic content, HTTPS-based APIs, and any other server-side code are hosted on Cloud Functions. There are two kinds of Cloud Functions:
+
+- HTTPS functions are reachable via a URL like `https://europe-west6-OUR-PROJECT-ID.cloudfunctions.net/getCaseStats`.
+- Firestore-triggered functions respond to events in the database, they can't be called directly.
+
+Our Cloud Functions code is in `app/server/functions/src`.
+
+Using a custom domain to invoke HTTPS functions isn't possible for us yet, since we run outside of `us-central1`, where Firebase Hosting Rewrites [aren't supported yet](https://github.com/firebase/firebase-tools/issues/842). However, the hostname of our API should be a detail that's invisible to our users and therefore unimportant.
+
+### Development and testing
+
+To test your Firebase code (security rules, Cloud Functions) in a local environment, we use Firebase's [Emulator Suite](https://firebase.google.com/docs/emulator-suite).
+
+To manually experiment with the emulator suite, run the following from the `functions` directory:
+
+```sh
+npm run serve
+```
+
+To experiment with the HTTPS endpoints you can use the [Postman](https://www.postman.com/product/api-client/) collection of requests in `app/server/WHO.postman_collection.json`.
+
+Our unit tests (`app/server/functions/**_spec.ts`) should be the main source of truth however; they use the emulators under the hood.
+
+To run our unit tests, run the following from the `server` directory:
+
+```sh
+npm run test
+```
+
+### Deploying
+
+#### Initial setup
+
+When setting up a new project, follow the instructions in [terraform/README.md](terraform/README.md) to...
+
+- Obtain Terraform credentials.
+- Create the project resources you'll be working with.
+
+Make sure to also take the steps listed in [Manual Setup](terraform/README.md#manual-setup) that create the Firebase resources.
+
+#### To update existing projects
+
+To update the whole project to the latest version, from the `app/server` folder, run:
+
+```sh
+firebase deploy --project=YOUR-PROJECT-ID
+```
+
+If you only want to update static assets, run:
+
+```sh
+firebase deploy --project=YOUR-PROJECT-ID --only hosting
+```
+
+You should always specify a `--project=YOUR-PROJECT-ID`. If you'd like you can
+use one a [project alias](https://firebase.google.com/docs/cli#project_aliases) instead of a project ID. If you forget to specify a project ID using `--project` Firebase will deploy to the `default` alias, which is aimed at our `who-mh-v2-dev1` project.
```

Comment:
`who-mh2-dev1`, I think you can remove the `mh-v2` reference as you're the only one that's using the project and it will be deleted soon anyway.